src/components/Version/Version.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { version } from '../../../package.json';
+import { IconButton, IconButtonProps } from '@mui/material';
+
+const Version = ({
+  color = 'primary',
+  disabled = false,
+  title = version,
+}: IconButtonProps) => {
+  return (
+    <>
+      <IconButton color={color} disabled={disabled}>
```

Comment:
Looks nice. I am wondering if we need to pick color in here. I believe black should do the trick. Or I am missing something?

And I think we need letter 'v' to reflect that it's a version.

Other than that LGTM.

Reply:
Thanks Domante, if shown on a black background, we can switch the color in the theme. Added the `v`.